auth/grpctransport/grpctransport.go

```diff
@@ -71,6 +71,10 @@ type Options struct {
 	// DetectOpts configures settings for detect Application Default
 	// Credentials.
 	DetectOpts *detect.Options
+	// UniverseDomain is the default service domain for a given Cloud universe.
+	// The default value is "googleapis.com". This is the universe domain
```

Comment:
don't mention client opts. But mention the difference between this ud and adc ud -- for validation only

Reply:
Done, thanks!